src/Wallabag/CoreBundle/Entity/Config.php

```diff
@@ -126,6 +126,51 @@ class Config
      */
     private $displayThumbnails;
 
+    /**
+     * @var string
+     *
+     * @ORM\Column(name="font", type="text", nullable=true)
+     *
+     * @Groups({"config_api"})
+     */
+    private $font;
+
+    /**
+     * @var float
+     *
+     * @ORM\Column(name="fontsize", type="float", nullable=true)
+     *
+     * @Groups({"config_api"})
+     */
+    private $fontsize;
+
+    /**
+     * @var float
+     *
+     * @ORM\Column(name="line_height", type="float", nullable=true)
+     *
+     * @Groups({"config_api"})
+     */
+    private $lineHeight;
+
+    /**
+     * @var float
+     *
+     * @ORM\Column(name="max_width", type="float", nullable=true)
+     *
+     * @Groups({"config_api"})
+     */
+    private $maxWidth;
+
+    /**
+     * @var string
+     *
+     * @ORM\Column(name="custom_css", type="text", nullable=true)
+     *
+     * @Groups({"config_api"})
```

Comment:
Should this be exposed to the API as it seems to be a calculated field? I don't think it'll be _usable_ for clients other than the web UI